src/main/java/com/uwetrottmann/tmdb2/TmdbInterceptor.java

```diff
@@ -79,7 +79,7 @@ public static Response handleIntercept(Chain chain, Tmdb tmdb) throws IOExceptio
                 try {
                     Integer retry = Integer.parseInt(retryAfter);
                     Thread.sleep((int) ((retry + 0.5) * 1000));
-                    response = chain.proceed(builder.build());
+                    return handleIntercept(chain,tmdb);
```

Comment:
This should not be called within the try/catch block as it may hide an exception thrown by `handleIntercept()`. I guess only parsing and sleeping should be guarded and if it fails it should return the existing response. Something like:

```java
if (retryAfter != null) {
    try {
        Integer retry = Integer.parseInt(retryAfter);
        Thread.sleep((int) ((retry + 0.5) * 1000));
    } catch (NumberFormatException | InterruptedException e) {
        return response; // failed to wait
    }
    return handleIntercept(chain, tmdb);
}
```